code/app/src/main/java/com/example/sigma_blue/DetailsFragment.java

```diff
@@ -59,12 +62,23 @@ public void onCreate(Bundle savedInstanceState)
         super.onCreate(savedInstanceState);
 
         // Load item from bundle
-        currentItem = new Item();
-        if (getArguments() != null)
-        {
-            currentItem = (Item)getArguments().getSerializable(ARG_ITEM);
-            oldItemID = getArguments().getString("id");
-        }
+//        currentItem = new Item();
```

Comment:
I would say that we should not keep placeholder code in the main commit, since they are marking us for code quality